src/include/platform/ConnectivityManager.h

```diff
@@ -177,19 +178,27 @@ class ConnectivityManager
     void ClearWiFiStationProvision();
     CHIP_ERROR GetAndLogWiFiStatsCounters();
 #if CHIP_DEVICE_CONFIG_ENABLE_WIFIPAF
-    struct WiFiPAFAdvertiseParam;
+    struct WiFiPAFAdvertiseParam
+    {
+        /* Frequency list */
+        ReadOnlyBuffer<uint16_t> freq_list;
+        /* publish_id */
+        uint32_t publish_id;
+    };
+
+    WiFiPAFAdvertiseParam mPafAdverParam;
+    CHIP_ERROR SetWiFiPAFAdvertisingEnabled(bool val);
+    CHIP_ERROR SetWiFiPAFPublishParam(ReadOnlyBufferBuilder<uint16_t> & knownFreqListBuilder);
```

Comment:
Why are we passinga `builder` to a setter? It seems to make more sense to pass in a `ReadOnlyBuffer &&` instead.

Reply:
Yes, I will change it. Thank you for the suggestion